package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@divriots/dockit-core",
-  "version": "0.1.8",
+  "version": "0.1.9",
   "description": "",
   "license": "ISC",
   "type": "module",
   "homepage": "{{ homepage }}",
   "scripts": {
     "serve": "dsd-cli serve 4BeMe20hqOWTkdUL2NuJ --open"
   },
+  "dependencies": {
+    "copy-to-clipboard": "^3.3.1",
+    "lodash": "^4.17.21",
+    "tailwindcss": "^2.2.15"
+  },
   "devDependencies": {
-    "dsd-cli": "latest"
+    "dsd-cli": "latest",
```

Comment:
Kinda unrelated, but could use the opportunity to remove the dependency and have the serve do **npx backlight@latest**

Reply:
Resolved :)